src/application/clients/crux-wallet-client.ts

```diff
@@ -230,27 +230,27 @@ export class CruxWalletClient {
         return {success, failures};
     }
 
-    @throwCruxClientError
-    public blacklistUsers = async (fullCruxIDs: string[]): Promise<{success: string[], failures: string[]}> => {
-        await this.initPromise;
-        const cruxUserWithKey = await this.getCruxUserByKey();
-        if (!cruxUserWithKey) {
-            throw ErrorHelper.getPackageError(null, PackageErrorCode.UserDoesNotExist);
-        }
-        const failures: string[] = [];
-        const registeredCruxUsers: string[] = [];
-        for (const fullCruxID of fullCruxIDs) {
-            const cruxUser = await this.getCruxUserByID(fullCruxID);
-            if (!cruxUser) {
-                failures.push(fullCruxID);
-            } else {
-                registeredCruxUsers.push(fullCruxID);
-            }
-        }
-        cruxUserWithKey.setBlacklistedCruxIDs(registeredCruxUsers);
-        await this.cruxUserRepository.save(cruxUserWithKey, this.getKeyManager());
-        return {success: registeredCruxUsers, failures};
-    }
+    // @throwCruxClientError
+    // public blacklistUsers = async (fullCruxIDs: string[]): Promise<{success: string[], failures: string[]}> => {
+    //     await this.initPromise;
```

Comment:
don't comment anything, just don't change anything that's present. your changes do not intersect.
earlier I had not merged sanchay's `blacklistUsers` feature branch

Reply:
I am removing all blacklist wala code

Reply:
done